test/test_masks.py

```diff
@@ -0,0 +1,29 @@
+from lensless.mask import CodedAperture, PhaseContour, FresnelZoneAperture
+
+
+def test_masks():
```

Comment:
let's do a separate test for each mask. For all of them (except PhaseContour), you can remove the call to `phase_retrieval` as this is only relevant for `PhaseContour`.

For `PhaseContour` we can add a test to make sure the simulated PSF (propagating the phase pattern) is close to the target amplitude (i.e. the output of the canny edge detector

Reply:
Rest will be primarily functional tests

Reply:
Are you sure? When I look at the PhlatCam article, I see an example of phase mask design applied on a FlatCam-like mask.

Reply:
Fig 8? When I find that Figure somewhat deceiving...because the caption says "phase mask design" but FlatCam is an amplitude mask.

In the original [FlatCam](https://arxiv.org/abs/1509.00116) paper, they don't apply any phase retrieval algorithm to obtain the mask pattern.

And for [FZA](https://www.nature.com/articles/s41377-020-0289-9), it's also an amplitude mask right? "The binary FZA, as a substitute of the GZP, consists of only transparent and opaque zones."